zmdocument.cls

```diff
@@ -207,33 +207,64 @@
 \chead{}
 \pagestyle{scrheadings}
 
+\newcommand{\addsubsec}[1]{
+   \subsection*{#1}
+   %\addxcontentsline{toc}{subsection}{#1}
+}
+
+\newcommand{\addsubsubsec}[1]{
+   \subsubsection*{#1}
+   %\addxcontentsline{toc}{subsubsection}{#1}
+}
+
+\newcommand{\addpar}[1]{
+   \paragraph*{#1}
+}
+
+\newcommand{\addsubpar}[1]{
+   \subparagraph*{#1}
+}
+
+
+\newcommand{\sectionningGenerator}[3]{
+   \newenvironment{#1}{
+      \csname#3\endcsname{#2}
+      \let\levelTwoTitle\addsubsec
+      \let\levelThreeTitle\addsubsubsec
+      \let\levelFourTitle\addpar
+      \let\levelFiveTitle\addsubparq
+   }{}
+}
+
+
 \DeclareOption{small}{
    \let\levelOneTitle\section
    \let\levelTwoTitle\subsection
    \let\levelThreeTitle\subsubsection
    \let\levelFourTitle\paragraph
    \let\levelFiveTitle\subparagraph
-   \newcommand{\levelOneIntroduction}{\addsec{Introduction}}
-   \newcommand{\levelOneConclusion}{\addsec{Conclusion}}
    \def\thesection{\arabic{section}}
    \def\thefigure{\thesection.\arabic{figure}}
    \def\thetable{\thesection.\arabic{table}}
    \def\theequationFloat{\thesection.\arabic{equationFloat}}
    \automark[subsection]{section}
+   \sectionningGenerator{LevelOneIntroduction}{Introduction}{addsec}
+   \sectionningGenerator{LevelOneConclusion}{Conclusion}{addsec}
```

Comment:
The value `Introduction` (that will be displayed into the PDF) should be a variable (with a default value to `Introduction`) that can be set by the user. This will allow non French/English user to use this template without editing this source.

This is the same for other `Conclusion`.